model/shared/src/main/scala/explore/model/util/NonEmptySetWrapper.scala

```diff
@@ -74,6 +77,9 @@ final class NonEmptySetWrapper[IdSet, Id](self: IdSet, iso: Iso[IdSet, NonEmptyS
   @inline
   def intersects(other: IdSet): Boolean = intersect(other).nonEmpty
 
+  @inline
+  def firstAndOnly: Option[Id] = if (selfSet.length === 1) selfSet.head.some else none
```

Comment:
`single` ?

Reply:
Sounds good. I couldn't think of a good name at the moment I created it....